firestore/integration_test_internal/src/aggregate_query_test.cc

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "firebase/firestore.h"
+#include "firestore_integration_test.h"
+
+#include "gtest/gtest.h"
+
+namespace firebase {
+namespace firestore {
+
+using AggrgegateQueryTest = FirestoreIntegrationTest;
+
+size_t AggregateQueryHash(const AggregateQuery& aggregate_query) {
+  return aggregate_query.Hash();
+}
+
+TEST_F(AggrgegateQueryTest, TestHashCode) {
```

Comment:
Please put all of the test methods into an anonymous namespace, to avoid polluting the firebase::firestore namespace. This includes the `using` declaration above.

e.g. https://github.com/firebase/firebase-cpp-sdk/blob/b1a5444b10c751c4c039a29bb9fe531b39d1b9ad/firestore/integration_test_internal/src/transaction_options_test.cc#L26

This comment also applies to aggregate_query_snapshot_test.cc